### PR TITLE
Moves HTTP/3 outside the experimental section

### PR DESCRIPTION
**Pull request by sdelicata**
### What does this PR do?

This PR moves HTTP/3 outside the experimental section.

### Motivation

HTTP/3 is stable

### More

- [ ] ~Added/updated tests~
- [x] Added/updated documentation

### CODE REVIEW

**Review by rtribotte — APPROVED** [review]
LGTM 👌

**Review by tomMoulard — APPROVED** [review]
LGTM :ok_hand:

File: docs/content/migration/v2-to-v3.md [hunk 1]
```diff
@@ -50,3 +50,7 @@ and should be explicitly combined using logical operators to mimic previous beha
 
 In v3, the `Content-Type` header is not auto-detected anymore when it is not set by the backend.
 One should use the `ContentType` middleware to enable the `Content-Type` header value auto-detection.
+
+## HTTP/3
+
+In v3, HTTP/3 is no longer an experimental feature. The `experimental.http3` option has been removed from the static configuration.
```
Comment (ldez) [inline docs/content/migration/v2-to-v3.md:56]:
```suggestion
In v3, HTTP/3 is no longer an experimental feature.
The `experimental.http3` option has been removed from the static configuration.
```

**Comment by mpl** [comment]
no-merge for now because alt-svc needs to be discussed further and taken into account.

File: docs/content/routing/entrypoints.md [hunk 2]
```diff
@@ -312,39 +312,31 @@ entryPoints:
...
-    
-    [entryPoints.name.http3]
-    ```
-    
-    ```bash tab="CLI"
-    --experimental.http3=true 
-    --entrypoints.name.http3
-    ```
+    As HTTP/3 uses UDP,
+    you can't have a TCP entryPoint with HTTP/3 on the same port as a UDP entryPoint.
```
Comment (mpl) [inline docs/content/routing/entrypoints.md:337]:
```suggestion
    As HTTP/3 actually uses UDP, when traefik is configured with a TCP entryPoint on port N with HTTP/3 enabled,
    the underlying HTTP/3 server that is started automatically listens on UDP port N too. As a consequence,
    it means port N cannot be used by another UDP entryPoint.
```

**Review by mpl — APPROVED** [review]

**Pull request merged**